packages/indexer/src/core/db.ts

```diff
@@ -188,18 +188,34 @@ export class DB {
   }
 
   async getMessages(req: MsgRequest): Promise<NomadMessage[]> {
-    const take = req.size || 15;
-    const page = req.page || 1;
-    const skip = (page || -1) * take;
+    const take = req.size ? parseInt(req.size) : 15;
+    const page = req.page ? parseInt(req.page) - 1 : 0;
```

Comment:
Are `req.size` and `req.page` strings here? Gotta be careful with `0` since it's truthy which can lead to a `-1` page value

Reply:
the page value is tested in the next line to be not less than 0
`if (page < 0) throw new Error(`Page is less than a 0`);`
Though I will go with first page start with 0 and not with 1